### PR TITLE
Quota

### PR DESCRIPTION
**Pull request by tinnh0c79**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -84,6 +84,13 @@ define a quota as follows:
 * `gold.storageclass.storage.k8s.io/requests.storage: 500Gi`
 * `bronze.storageclass.storage.k8s.io/requests.storage: 100Gi`
 
+Quota support for local ephemeral storage was added as an alpha feature in release 1.8.
```
Comment (sftim) [inline docs/concepts/policy/resource-quotas.md]: Could this use a [`feature-state` shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state)?

**Comment by DanyC97** [comment]
@Phamhien77779 are you still planning to work on this PR ?

**Comment by sftim** [comment]
As I understand it, `LocalStorageCapacityIsolation` feature-gate was removed in Kubernetes v1.9
If that's right, should // can this PR be closed?

**Comment by poothia** [comment]
@Phamhien77779  please resolve the merge conflict.

**Pull request closed without merging**